crates/neon-sys/native/binding.gyp

```diff
@@ -17,6 +17,7 @@
             'Debug': {
                 'msvs_settings': {
                     'VCCLCompilerTool': {
+                        'WholeProgramOptimization': 'false',
```

Comment:
Given how painful it was to discover this, might be worth a comment explaining this is necessary for Node versions < x.y on Windows, maybe with a link to the issue.